packages/bpk-component-popover/stories.scss

```diff
@@ -6,7 +6,7 @@
 
 .bpk-popover-spacer {
   display: flex;
-  height: 500 * $bpk-one-pixel-rem;
+  height: 100 * $bpk-one-pixel-rem;
```

Comment:
this change is because the "switch theme" button was outside the viewport in the storybook, 500 look too big anyway so I changed it